.travis.yml

```diff
@@ -26,59 +33,43 @@ matrix:
         # Do a coverage test in Python 3.
         - python: 3.5
           env: SETUP_CMD='test --coverage'
-        # Check for sphinx doc build warnings - we do this first because it
-        # may run for a long time. 
-        # Update: ignore warnings, just check for errors
+        # Check for sphinx doc build errors
         - python: 3.5
           env: SETUP_CMD='build_sphinx'
 
```

Comment:
I thought we might as well switch the docs build to 3.5, but I see we are getting some bogus distutils error on that now. Grrr.